docs/quickstart.md

```diff
@@ -1,100 +1,120 @@
+- Authors (Optional): Enter the owner of this SubQuery project here
+- Description (Optional): You can provide a short paragraph about your project that describe what data it contains and what users can do with it
+- Version (Required): Enter a custom version number or use the default (`1.0.0`)
+- License (Required): Provide the software license for this project or accept the default (`Apache-2.0`)
 
-Answer questions according to the guidelines， then you should see a folder with your project name has been created inside the directory, you can use this as the start point of your project. And the files should be identical as in the [Directory Structure](/directory_structure).
+After the initialisation process is complete, you should see a folder with your project name has been created inside the directory. The files and  should be identical as in the [Directory Structure](/create/directory_structure).
 
-Last, under the project directory, run following command to install all the dependency.
-```
-yarn install
-```
+Last, under the project directory, run following command to install the new project's dependencies.
+```shell
+cd PROJECT_NAME
 
+# NPM
+npm install
 
-## Configure your project
+# Yarn
+yarn install
+```
 
-In the starter package, we have provided a simple example of project configuration. You will be mainly working on the following files:
+## Configure and Build the Starter Project
 
+In the starter package that you just initialised, we have provided a standard set of  configuration for your new project. You will mainly be working on the following files:
 - The Manifest in `project.yaml`
 - The GraphQL Schema in `schema.graphql`
 - The Mapping functions in `src/mappings/` directory
 
-For more information on how to write the SubQuery, 
-check out our doc section on [Define the SubQuery](/define_a_subquery) 
+For more information on how to write the SubQuery, check out our documentation under [Define the SubQuery](/create/define_a_subquery) 
 
-#### Code generation
+### GraphQL Model Generation
 
-In order to index your SubQuery project, it is mandatory to build your project first.
-Run this command under the project directory.
+In order to index your SubQuery project, you must first generate the required GraphQL models that you have defined in your GraphQL Schema file (`schema.graphql`). Run this command in the root of the project directory.
+```shell
+# NPM 
+npm run-script codegen
 
-````
+# Yarn
 yarn codegen
-````
+```
 
-## Build the project
+You'll find the generated models in the `/src/types/models` directory
 
-In order to deploy your SubQuery project to our hosted service, it is mandatory to pack your configuration before upload.
-Run pack command from root directory of your project will automatically generate a `your-project-name.tgz` file.
+## Build the Project
 
-```
+In order run your SubQuery Project on a SubQuery node or our hosted service, you need to build and pack your work.
```

Comment:
This instruction is out of date, the pack command is still available but not a requirement of our hosted service.

Reply:
Is this fine?